runtime/master/src/main/java/org/apache/nemo/runtime/master/scheduler/LocalitySchedulingConstraint.java

```diff
@@ -78,7 +78,10 @@ private LocalitySchedulingConstraint(final BlockManagerMaster blockManagerMaster
           .map(handler -> {
             try {
               return handler.getLocationFuture().get();
-            } catch (InterruptedException | ExecutionException e) {
+            } catch (ExecutionException e) {
```

Comment:
`final ExecutionException e`?

Reply:
I think this can be handled with a different PR. I believe most of the exceptions are non-final at the moment.

Related discussion: https://stackoverflow.com/a/3516852